packages/mobile/src/components/WarningModal.tsx

```diff
@@ -0,0 +1,80 @@
+import TextButton from '@celo/react-components/components/TextButton'
```

Comment:
Copied over from DataSaver with small changes to allow for one or two buttons. Note that the redesign has not been applied- leaving that for a future PR